i18n.go

```diff
@@ -91,6 +91,14 @@ func loadMessages(path string) {
 	if error := filepath.Walk(path, loadMessageFile); error != nil && !os.IsNotExist(error) {
 		ERROR.Println("Error reading messages files:", error)
 	}
+
+	// Read in messages from the modules
+	for _, module := range Modules {
```

Comment:
It may be better to add a `MessagesPaths` similar to `TemplatePaths` in `revel.go` instead of accessing Modules directly. Thoughts? It's a minor difference, but more consistent with the current module convention.